### PR TITLE
Add rate_track

### PR DESCRIPTION
**Pull request by JasonLG1979**
Adds the ability to rate a track. Feedback Id's are not needed with the new API.

### CODE REVIEW

File: pandora.py [hunk 1]
```diff
@@ -165,3 +172,24 @@ def __init__(self):
...
+
+    async def rate_track(self, track: Track, track_rating: TrackRating) -> None:
+        """
+        Rate a track.
+
+        :param track: The track to be rated.
+        :param track_rating: The new TrackRating.
+        """
+        if track_rating is TrackRating.NONE:
+            if track.rating is not TrackRating.NONE:
```
Comment (TingPing) [inline pandora.py:194]: Wonder if it makes sense to just always send it since it is possible the information we have is outdated?

Comment (JasonLG1979) [reply pandora.py:194]: The info we have shouldn't be outdated. You're only *suppose to* be able to listen from one device/app per account at a time.

Comment (TingPing) [reply pandora.py:194]: Alright, you'll need to rebase both PRs btw.

Comment (JasonLG1979) [reply pandora.py:194]: I know. I've got a lot coming. I'll rebase as you merge them.

File: pandora.py [hunk 2]
```diff
@@ -11,6 +11,13 @@ class AudioFormat(enum.Enum):
     AACPLUS = 'aacplus'
 
 
+class TrackRating(enum.IntEnum):
```
Comment (TingPing) [inline pandora.py:14]: Same comment about this could be `enum.Enum` since its never used as an `int`

Comment (JasonLG1979) [reply pandora.py:14]: On this one I do use the int.

See:
https://github.com/TingPing/pandora-rest/pull/14/commits/bb7189efc8fa6154b3b9f5e0b2d26d16806f6324#diff-c1ec1e0b2d287fbebda0b1d01535e36fR74

Comment (JasonLG1979) [reply pandora.py:14]: A new song in a playlist will start as either 0 or 1 as a rating.

Comment (TingPing) [reply pandora.py:14]: Alright.

**Review by TingPing — APPROVED** [review]
Feel free to merge once rebased.

**Comment by JasonLG1979** [comment]
right on.

**Pull request merged**